properties/basic.py

```diff
@@ -760,7 +762,7 @@ def unicode(self, value):
 
     @property
     def regex(self):
-        """Regular expression the string must comply with"""
+        """Regular expression the string must include"""
```

Comment:
I think maybe the first is more accurate. It's just that `match` implicitly requires that the match happens at the beginning of the string (i.e., `^` is prepended to the expression). So, the second is actually more generic.

That said, I am wondering if there are performance benefits to `match` over `search`. We may want to profile it at some point (e.g., if it cuts down on backtracking steps).